modules/vba_documents/spec/workers/report_monthly_submissions_spec.rb

```diff
@@ -1,62 +1,9 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'date'
-require './modules/vba_documents/spec/support/vba_document_fixtures'
-require './modules/vba_documents/lib/vba_documents/sql_support'
 
 RSpec.describe VBADocuments::ReportMonthlySubmissions, type: :job do
-  include VBADocuments::Fixtures
-
-  before { Timecop.freeze(Time.zone.parse('2021-04-8 00:00:00 UTC')) }
-
-  after { Timecop.return }
-
   describe '#perform' do
-    include VBADocuments::SQLSupport
-
-    base_fixture_path = 'reports/monthly_report'
-    monthly_counts = "#{base_fixture_path}/monthly_counts.yml"
-    summary = "#{base_fixture_path}/monthly_summary.yml"
-    still_processing = "#{base_fixture_path}/still_processing.yml"
-    still_success = "#{base_fixture_path}/still_success.yml"
-    avg_times = "#{base_fixture_path}/avg_processing_time.yml"
-    final_monthly_results = "#{base_fixture_path}/final_monthly_results.yml"
-    mode = "#{base_fixture_path}/mode.yml"
-    max_avg = "#{base_fixture_path}/max_avg.yml"
-    rolling_elapsed_times = "#{base_fixture_path}/rolling_elapsed_times.yml"
```

Comment:
I'm going to remove and replace these fixtures in the next PR, but this PR was already too big to do so.